controllers/warehouseController.js

```diff
@@ -61,6 +61,18 @@ const updateWarehouseDetails = async (req, res) => {
   }
 };
 
+
+const getAllWarehouses = async(req, res) => {
+  try {
+    const allWarehouses = await knex("warehouses");
+    res.status(200).json(allWarehouses);
```

Comment:
we can ensure that the client is aware that no warehouses exist. by retuning a res with more information